src/app.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import Home from './home'
+
+export default class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      cardInfo: [
+        {
+          id: 0,
+          question: ' ',
+          answer: ' '
+        }
+      ],
+      view: {
+        path: '',
+        params: ' '
+      },
+      lastId: 0
+    }
+    this.handleSave = this.handleSave.bind(this)
+  }
+  componentDidMount() {
```

Comment:
Please do not include code you are not using. Even if you think you need it later.